src/apps/frontend/pages/authentication/otp/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import toast from 'react-hot-toast';
+import { useNavigate } from 'react-router-dom';
+
+import constants from '../../../constants/routes';
+import { AsyncError } from '../../../types';
+import useTimer from '../../../utils/use-timer.hook';
+import AuthenticationFormLayout from '../authentication-form-layout';
+import AuthenticationPageLayout from '../authentication-page-layout';
+
+import OTPForm from './otp-form';
+
+export const OTPPage: React.FC = () => {
+  const { startTimer, remaininingSecondsStr, isResendEnabled } = useTimer();
+
+  const navigate = useNavigate();
+
+  const onSuccess = () => {
+    toast.success('Logged in success');
+    navigate(constants.DASHBOARD);
+  };
+
+  const onResendSuccess = () => {
+    startTimer();
+    toast.success('OTP resent');
```

Comment:
This is quite a poor copyright, and seems like we are still not paying attention to these messages. Can you please use Generate AI, Google and other online resources to craft better messages?